src/views/Schedule2.vue

```diff
@@ -88,4 +114,23 @@ export default Vue.extend({
     width: calc(100% - 20px);
   }
 }
+
+.now-bar {
+  position: absolute;
+  width: 100%;
+  height: 3px;
+  background-color: black;
+  z-index: 2;
+}
```

Comment:
We should stay close to https://brand.engineering.publicissapient.fr/colors.php; I suggest:
```suggestion
.now-bar {
  position: absolute;
  width: 100%;
  border-top: 2px dashed $tertiary;
  z-index: 2;
}
```

Reply:
Fixed